recipes/cuda-pathfinder/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+context:
+  name: cuda-pathfinder
+  version: "1.0.0"
+  # no-arch packages can't use selectors, so we avoid using the "match(python, "<3.9")" selector this way
+  python_min: "3.9"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  # TODO: switch to use the stable release url in the next release
+  url: https://github.com/nvidia/cuda-python/archive/${{ name }}-v${{ version }}.tar.gz
+  sha256: 98820d855f06ec23828669196d2d21e310095809dd89ba6d32b875637a1d9656
+
+build:
+  noarch: python
+  script: cd cuda_pathfinder && python -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pip
+    - setuptools
+  run:
+    - python >=${{ python_min }}
+    - cuda-version >=11.0
```

Comment:
Do we want to put an upper pin on this? I.E. `<13`?

Reply:
To refine what I said in the meeting, I still believe:
* that we can make the `pathfinder` highly backward compatible,
* but in the meantime I found out that we have to make a new `pathfinder` release along with new CTK releases (maybe not all of them, but probably some, certainly the next one).

So at the moment `<13` actually makes sense.